kong/cmd/utils/dnsmasq_signals.lua

```diff
@@ -71,7 +71,7 @@ end
 
 function _M.stop(kong_config)
   log.verbose("stopping dnsmasq at %s", kong_config.dnsmasq_pid)
-  return kill(kong_config.dnsmasq_pid, "-9")
+  return kill(kong_config.dnsmasq_pid, "-15") -- SIGTERM
```

Comment:
Okay cool 

> On Jul 5, 2016, at 7:23 PM, Marco Palladino <notifications@github.com> wrote:
> 
> In kong/cmd/utils/dnsmasq_signals.lua:
> 
> > @@ -71,7 +71,7 @@ end
> >  
> >  function _M.stop(kong_config)
> >    log.verbose("stopping dnsmasq at %s", kong_config.dnsmasq_pid)
> > -  return kill(kong_config.dnsmasq_pid, "-9")
> > +  return kill(kong_config.dnsmasq_pid, "-15") -- SIGTERM
> When Dnsmasq is terminated with SIGTERM, because Dnsmasq handles the PID itself, it by default also deletes it.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>